Model/Datasource/Oracle.php

```diff
@@ -552,7 +557,7 @@ function listSources($data = null) {
 /**
  * Returns an array of the fields in given table name.
  *
- * @param object $model instance of a model to inspect
+ * @param Model|string $model instance of a model to inspect
```

Comment:
I think our code will fail if $model is of type string, because of:
```
 		if (!empty($model->sequence)) {
 			$this->_sequenceMap[$tableOnly] = $model->sequence;
 		} elseif (!empty($model->table)) {
 			if ($this->sequenceExists($model->table . '_seq', $tableSchema)) {
 				$this->_sequenceMap[$tableOnly] = $model->table . '_seq';
 			} else {
 				$this->_sequenceMap[$tableOnly] = false;
 			}
 		}
```

Suggestions?